waspc/cli/src/Wasp/Cli/Command/Update.hs

```diff
@@ -0,0 +1,22 @@
+module Wasp.Cli.Command.Update
+  ( update,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import System.Exit (ExitCode (..), exitWith)
+import System.Process (system)
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import qualified Wasp.Cli.Common as Common
+import qualified Wasp.Message as Msg
+
+update :: Command ()
+update = do
+  cliSendMessageC $ Msg.Start "Updating wasp..."
+  status <- liftIO $ system "curl -sSL https://get.wasp-lang.dev/installer.sh | sh"
```

Comment:
This call to `system`, will it forward STDIO/STDERR to the STDIO/STDERR of our haskell program? It would be cool if it would! That way they can see progress, and possibly any errors.

Reply:
Yes it does :)